spec/introduction.md

```diff
@@ -16,10 +16,22 @@ We need answers to a simple question:
 
 > Does `Entity X` have `Authorization Y`, in the context of `Ecosystem Governance Framework Z`?
 
-The Trust Registry Quert Protocol (TRQP) serves to provide a simple interface to enable querying of systems of record that provide the information that drives a trust registry. There are a plethora of systems that contain answers that are required to make trust decisions. The protocol is intended to make the communication with any particular system-of-record consistent and simple.
+The Trust Registry Quert Protocol (TRQP) serves to provide a simple suite of adapted interfaces to enable querying of systems of record that provide the information that drives a trust registry. There are a plethora of systems that contain answers that are required to make trust decisions. The protocol is intended to make the communication with any particular system-of-record consistent and simple.
+
+::: todo
+**we should explain the "systems of record" content at high level**
```

Comment:
Maybe even before this, define what the minimum requirements of a "record" is, at a model level.

Reply:
"record" or "system of record"?